src/datashare/azext_datashare/generated/commands.py

```diff
@@ -0,0 +1,215 @@
+    cf_consumer_source_data_set,
+    cf_synchronization_setting,
+    cf_trigger,
+)
+
+
+datashare_share = CliCommandType(
+    operations_tmpl='azext_datashare.vendored_sdks.datashare.operations._shares_operations#SharesOperations.{}',
+    client_factory=cf_share,
+)
+
+
+datashare_account = CliCommandType(
+    operations_tmpl='azext_datashare.vendored_sdks.datashare.operations._accounts_operations#AccountsOperations.{}',
+    client_factory=cf_account,
+)
+
+
+datashare_consumer_invitation = CliCommandType(
+    operations_tmpl='azext_datashare.vendored_sdks.datashare.operations._consumer_invitations_operations#ConsumerInvitationsOperations.{}',
+    client_factory=cf_consumer_invitation,
+)
+
+
+datashare_consumer_source_data_set = CliCommandType(
+    operations_tmpl='azext_datashare.vendored_sdks.datashare.operations._consumer_source_data_sets_operations#ConsumerSourceDataSetsOperations.{}',
+    client_factory=cf_consumer_source_data_set,
+)
+
+
+datashare_data_set = CliCommandType(
+    operations_tmpl='azext_datashare.vendored_sdks.datashare.operations._data_sets_operations#DataSetsOperations.{}',
+    client_factory=cf_data_set,
+)
+
+
+datashare_data_set_mapping = CliCommandType(
+    operations_tmpl=(
+        'azext_datashare.vendored_sdks.datashare.operations._data_set_mappings_operations#DataSetMappingsOperations.{}'
+    ),
+    client_factory=cf_data_set_mapping,
+)
+
+
+datashare_email_registration = CliCommandType(
+    operations_tmpl='azext_datashare.vendored_sdks.datashare.operations._email_registrations_operations#EmailRegistrationsOperations.{}',
+    client_factory=cf_email_registration,
+)
+
+
+datashare_invitation = CliCommandType(
+    operations_tmpl=(
+        'azext_datashare.vendored_sdks.datashare.operations._invitations_operations#InvitationsOperations.{}'
+    ),
+    client_factory=cf_invitation,
+)
+
+
+datashare_provider_share_subscription = CliCommandType(
+    operations_tmpl='azext_datashare.vendored_sdks.datashare.operations._provider_share_subscriptions_operations#ProviderShareSubscriptionsOperations.{}',
+    client_factory=cf_provider_share_subscription,
+)
+
+
+datashare_share_subscription = CliCommandType(
+    operations_tmpl='azext_datashare.vendored_sdks.datashare.operations._share_subscriptions_operations#ShareSubscriptionsOperations.{}',
+    client_factory=cf_share_subscription,
+)
+
+
+datashare_synchronization_setting = CliCommandType(
+    operations_tmpl='azext_datashare.vendored_sdks.datashare.operations._synchronization_settings_operations#SynchronizationSettingsOperations.{}',
+    client_factory=cf_synchronization_setting,
+)
+
+
+datashare_trigger = CliCommandType(
+    operations_tmpl='azext_datashare.vendored_sdks.datashare.operations._triggers_operations#TriggersOperations.{}',
+    client_factory=cf_trigger,
+)
+
+
+def load_command_table(self, _):
+
+    with self.command_group('datashare', datashare_share, client_factory=cf_share, is_experimental=True) as g:
```

Comment:
If you don't want this command group to be exprimental, just remove `is_expermimental=True` tag here.

Since this code is generated, you need to modify the readme file used by code gen. @kairu-ms for guidance